testsuite/features/step_definitions/command_steps.rb

```diff
@@ -384,7 +384,7 @@
 When(/^I wait until the channel "([^"]*)" has been synced$/) do |channel|
   begin
     repeat_until_timeout(timeout: 7200, message: 'Channel not fully synced') do
-      _result, code = $server.run("test -f /var/cache/rhn/repodata/#{channel}/repomd.xml", check_errors: false)
+      _result, code = $server.run("test -f /var/cache/rhn/repodata/#{channel}/products.xml", check_errors: false)
```

Comment:
maybe we should add just a one line comment mentioning that's the last file to be synced during a reposync? :thinking: just so it's understandable in the future why we're looking for this one and not the `repomd.xml`, which is the file that actually matters but not the last one to sync

Reply:
Agree!

Reply:
> Maybe you want to ckeck, if no `.new` files exists. On a re-sync, the old files stay, but the new generated files are created with this suffix until all files are ready

Great info, thank you!

Reply:
Do you want to add it to this PR or do we leave it as is for now @srbarrios?

Reply:
Added